docs/commands.md

```diff
@@ -104,11 +105,12 @@ The complexity of `TS.ADD` is always O(M) when M is the amount of compaction rul
 Append new samples to a list of series.
 
 ```sql
-TS.MADD key timestamp value [key timestamp value ...]
+TS.MADD [APPEND_ONLY] key timestamp value [key timestamp value ...]
```

Comment:
@ashtul I don't think we need it on MADD, we don't have the other attributes either like UNCOMPRESSED

Reply:
@gkorland IIUC because we don't have any flags in `MADD` adding one would be weird. on the other hand not adding it leads to other incompatible API.

Reply:
@gkorland @danni-m 
The UNCOMPRESSED option is for the `create` option in TS.ADD which isn't available for TS.MADD.
I agree it is a bit weird but otherwise, there is no option to prevent OOO

Reply:
It can also create weird parsing issues in case the user key is `APPEND_ONLY`, I would have avoid it on MADD